frontend/rolecall/src/app/cast/cast-editor-v2.component.html

```diff
@@ -1,34 +1,48 @@
 <app-loading-spinner *ngIf="!dataLoaded"></app-loading-spinner>
-<div [ngStyle]="dataLoaded ? {} : { 'display' : 'none' }" class="root-container">
+<div class="root-container"
+    [ngStyle]="dataLoaded ? {} : { 'display' : 'none' }">
   <div class="cast-selector-container">
     <div class="app-card card-margins left-card">
-      <div ngbDropdown #pieceDropdown="ngbDropdown">
-        <div class="app-card piece-select" ngbDropdownToggle>
-          <h1 class="left-card-title">
+      <div ngbDropdown
+          #pieceDropdown="ngbDropdown">
+        <div ngbDropdownToggle
+            class="app-card piece-select">
+          <h1 class="left-card-title"
+              [ngStyle]="{'font-style': selectedPiece && selectedPiece.siblingId > 0 ? 'italic' : 'normal' }">
```

Comment:
This could be a little more readable as a dynamic class instead of `ngStyle`, something like:
`[class.italic-font]="selectedPiece && selectedPiece.siblingId > 0"`

Reply:
An interesting point. Personally, I prefer using [ngStyle] to [ngClass] or [clsss.xxx] because I can see right there what is happening without having to go and look it up in another file. If we can create some overarching structure to our scss files, I could be persuaded to switch to [class.xxx]. (I would like to have one central file with all 'standard style classes. These classes should have a standard prefix so they are easily identifiable, etc. If you agree that we should do that across the board, I can change this right now.)

Reply:
ngStyle can be useful in complex cases, but the motivation here is to reduce the amount of logic in templates especially for simple cases with a clear alternative.

Looks like there's currently already a global scss file called `styles.scss`, maybe this global class `italic-font` can go in there for now and maybe it'll evolve to a slightly different structure in the future if needed? Though let's not make any broader changes as part of the same PR so as not to increase its scope.

Reply:
After reading the similar comment below, I decided to start building a standard library of style classes.

Suggestion implemented.